Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -34,12 +31,35 @@ def setUp(self):
             self.workspace_name = config["workspace_name"]
             self.api_key = config["api_key"]
             self.alert_id = config["alert_id"]
+            self.tenant_id = config["tenant_id"]
```

Comment:
Can you please update cohesity.json to include all these new values?
1. Also, please provide reasonable defaults for these values. For example, resource_url and scope should be same most of the time.
Also, please provide a version of cohesity.json which should be usable directly. I'll tell you a location where you can upload it internally offline.

Reply:
done.
wait for your location to upload.
resource_url and scope are 99% identical, but a little different, so still have to be 2 values.